.devcontainer/devcontainer.json

```diff
@@ -1,31 +1,38 @@
 {
-	"name": "relayrunner",
-	"remoteUser": "runner",
-	"image": "johee/akdc-kind:nodejs",
-	"runArgs": [
-		"--init",
-		"--privileged"
-	],
+	"name": "RelayRunner",
+	"dockerFile": "Dockerfile",
+
+	// do not change these if you want Docker in Docker support
+	"runArgs": [ "--init", "--privileged" ],
+	"mounts": [ "source=/var/run/docker.sock,target=/var/run/docker-host.sock,type=bind" ],
 	"overrideCommand": false,
-	"settings": { },
+
+	// some base images require a specific user name
+	"remoteUser": "vscode",
+
+	// set container specific settings
+	"settings": {
+		"terminal.integrated.defaultProfile.linux": "zsh"
+	},
+
 	"forwardPorts": [
 		30000,
 		30080,
 		30088,
 		32000,
 		32080,
-    32088
+		32088
 	],
 
 	// add labels
 	"portsAttributes": {
+		"6443": {"label": "k3d" },
 		"30000": { "label": "Prometheus" },
 		"30080": { "label": "Ngsa-Memory" },
 		"30088": { "label": "LodeRunner" },
 		"32000": { "label": "Grafana" },
 		"32080": { "label": "RelayRunner-Client" },
 		"32088": { "label": "RelayRunner-Backend" },
-		"30081": { "label": "Kind"}
 	},
 
```

Comment:
If you know which port we might publish the app and the API to in the cluster, we could predefined them here even if we aren't quite at that point.  At least it would give us a placeholder for RelayRunner and the API.